app/models/application_record.rb

```diff
@@ -2,4 +2,22 @@
 
 class ApplicationRecord < ActiveRecord::Base
   primary_abstract_class
+
+  before_save :set_updated_at
+  before_create :set_inserted_at
```

Comment:
I'm wondering why this was added on this PR. It definitely needs some tests since it is changing default Rails behavior. I think the `inserted_at` column makes sense, but Rails already uses `updated_at` column on the model so I don't think that part is necessary? I did some searching for this issue and found this stackoverflow post. What do you think of [this solution](https://stackoverflow.com/a/13457972) which adds the new column to the `timestamp_attributes_for_create` class method instead? Seems a bit cleaner than creating a new before_save callback (and turning off the existing one).